src/Symfony/Component/CssSelector/XPath/Extension/ExtensionInterface.php

```diff
@@ -60,6 +60,13 @@ public function getPseudoClassTranslators(): array;
      */
     public function getAttributeMatchingTranslators(): array;
 
+    /**
+     * Returns relative combinators translators.
+     *
+     * @return callable[]
+     */
+    public function getRelativeCombinationTranslators(): array;
```

Comment:
this must be added using `@method` as adding a new method in an interface is a BC break